src/installing-workflow/configuring-object-storage.md

```diff
@@ -89,6 +90,20 @@ The slugrunner reads credentials from the below locations on the filesystem.
 - Key: `/var/run/secrets/object/store/access-key-id`
 - Secret: `/var/run/secrets/object/store/access-key-secret`
 
+## [deis/controller](https://github.com/deis/controller)
+
+When the controller needs to launch a new buildpack application or scale one up, it uses a [replication controller](http://kubernetes.io/docs/user-guide/replication-controller/). Since the slugrunner needs to download the slug to run, it needs the object storage location of the slug and the object storage credentials.
+
+### Environment Variables
+
+The controller needs no environment variables for object storage configuration.
+
+### Credentials
+
+Since the object storage location information comes from the builder, the controller only needs access to the credentials information. The controller gets this information by accessing the `minio-user` secret directly from the Kubernetes API.
```

Comment:
I'm starting to understand where some of my confusion arose from yesterday.  I think I had viewed the steps for _not_ using minio as starting with step 1: rip out everything minio-related... in reality, I guess step 1 (currently) is: pretend you're using minio and put your s3 (or whatever) creds where the minio creds usually go.  This is enlightening.